contrib/docker/docker-compose.yml

```
@@ -33,11 +31,25 @@ services:
     # instance expose the TLS port directly:
     ports:
       - 8448:8448/tcp
-    # ... or use a reverse proxy, here is an example for traefik:
+    # ... or use a reverse proxy, here is an example for traefik (you may disable ports):
```

I don't understand this, either.

```suggestion
    # ... or use a reverse proxy, here is an example for traefik:
```

---

I meant, 'ports' above (8448 as shown) are not necessary, if we use such traefik config. We use traefik to route all via proxy generally. If we stay with  "- ports 8448:8448" we could connect directly to Docker host and published port also.